control-plane-operator/controllers/hostedcontrolplane/hostedcontrolplane_controller.go

```diff
@@ -1224,6 +1224,17 @@ func (r *HostedControlPlaneReconciler) reconcile(ctx context.Context, hostedCont
 	// TODO: consider using a side container in the etcd pod instead.
 	r.Log.Info("Reconciling etcd-backup cronJob")
 	if hostedControlPlane.Spec.Platform.Type == hyperv1.AWSPlatform {
+		configMapName := "etcd-backup-config" // TODO: get configMap name from annotation?
+		configMap := &corev1.ConfigMap{}
+		if err := r.Client.Get(ctx, client.ObjectKey{Namespace: hostedControlPlane.Namespace, Name: configMapName}, configMap); err != nil {
+			// skip creating the cronJob if the configMap doesn't exist.
+			if apierrors.IsNotFound(err) {
+				return nil
```

Comment:
This may cause some confusion in the future. If we add other stuff to the reconcile function, it won't be immediately clear why it's not being called. I would enclose the reconcile of the backup in an if block and not return nil here.

Reply:
fixed